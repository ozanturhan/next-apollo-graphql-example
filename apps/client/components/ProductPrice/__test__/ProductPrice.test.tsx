import React from 'react';
import 'jest-styled-components';
import { render } from '@testing-library/react';
import { ProductPrice } from '../ProductPrice';

it('renders correctly', () => {
  const tree = render(<ProductPrice>1,234,00</ProductPrice>).container;
  expect(tree).toMatchSnapshot();
});
