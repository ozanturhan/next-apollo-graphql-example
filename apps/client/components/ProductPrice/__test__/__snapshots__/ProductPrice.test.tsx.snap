// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
.c0 {
  border-radius: 10px;
  background-color: #f4f4f4;
  padding: 15px;
  font-size: 1.3rem;
  font-weight: bold;
  margin-bottom: 15px;
}

<div>
  <span
    class="c0"
  >
    1,234,00
  </span>
</div>
`;
